Makefile

```diff
@@ -1,28 +1,27 @@
-all: vendor
+.PHONY: vendor all reset-cni
+.PHONY: clean clean-bins clean-rootfs clean-image clean-ssh-keys 
+
+all:
 	go build -o cni-noop ./cmd/cni-noop
 	go build -o cnispawn ./cmd/cnispawn
 	go build -o kubeadm-nspawn ./cmd/kubeadm-nspawn
 
+vendor: glide.lock | glide
+	glide --quiet install --strip-vendor
+glide.lock: | glide.yaml glide
```

Comment:
Sort of, yes. If

```
$ make vendor
make: *** No rule to make target 'glide.yaml', needed by 'glide.lock'.  Stop.
```

is what you want. The glide error is more descriptive though IMO:

```
[ERROR] Failed to find glide.yaml file in directory tree: Cannot resolve parent of /
```

i.e you could just remove `glide.yaml` here.